scientific/Detrend_Data/Detrend_data.ipynb

```diff
@@ -110,10 +110,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Removing Annual Cycle \n",
+    "## Removing the Annual Cycle \n",
     "[Back to Top](#top)\n",
     "\n",
-    "First we will remove the annual cycle for these **monthly** data. Because it is masked order matter as we will demo."
+    "First we will remove the annual cycle for these **monthly** data. Because it is masked (i.e. some data is missing) the order of operation matters, as we will demonstrate."
    ]
```

Comment:
@dnadeau4 I made a similar change to clarify. I also made several more text edits to endeavor to clarify things a bit more. This version is not the final, but it is worth merging this version into master since it is passable and we need to get the .png file into master so the page can be accessed on the tutorials page.